libraries/AP_BattMonitor/AP_BattMonitor_ESC.cpp

```diff
@@ -92,4 +93,15 @@ void AP_BattMonitor_ESC::read(void)
     }
 }
 
+bool AP_BattMonitor_ESC::reset_remaining(float percentage)
+{
+    delta_mah = 0.0f;
```

Comment:
Probably unneeded since you will already assign below, and if the reset fails we don't need to clear this (we should preserve previous state)

Reply:
I think its needed to get the actual values from the ESCs rather than a modified value?

Reply:
Oh, yeah I'm sorry. I see why you need it.